widgets/src/components/TaskList/TaskListContainer.js

```diff
@@ -29,36 +29,43 @@ class TaskList extends React.Component {
     columns: [],
     rows: [],
     size: 0,
-    currentPage: 0,
     connection: {},
     error: null,
     errorAlert: null,
+    lastPage: false,
   };
 
   timer = { ref: null };
 
   componentDidMount = async () => {
     const { lazyLoading, serviceUrl, pageCode, frameId } = this.props;
-    const { currentPage } = this.state;
 
-    SERVICE.URL = serviceUrl;
+    if (!LOCAL) {
+      // set the PDA domain to the URL passed via props
+      DOMAINS.PDA = serviceUrl;
```

Comment:
maybe serviceUrl should be passed to makeRequest and the correct domain chosen there?
also, what's our approach to mutation? I try to avoid it

Reply:
lez keep that for now, we still don't know for sure if the serviceUrl will be passed via props, so we can revisit it later.